$NetBSD: patch-mf,v 1.2 2012/05/07 03:22:36 sbd Exp $

Add DragonFly support.
Make sure nss libraries have a run path defined.

Apply attachment 543716 from
https://bugzilla.mozilla.org/show_bug.cgi?id=661609
to fix building on Linux 3.0 and up.

--- security/coreconf/config.mk.orig	2012-03-06 14:45:43.000000000 +0000
+++ security/coreconf/config.mk
@@ -63,7 +63,7 @@ endif
 #######################################################################
 
 TARGET_OSES = FreeBSD BSD_OS NetBSD OpenUNIX OS2 QNX Darwin BeOS OpenBSD \
-              AIX RISCOS WINNT WIN95 WINCE
+              AIX RISCOS WINNT WIN95 WINCE DragonFly Linux
 
 ifeq (,$(filter-out $(TARGET_OSES),$(OS_TARGET)))
 include $(CORE_DEPTH)/coreconf/$(OS_TARGET).mk
@@ -207,3 +207,4 @@ endif
 DEFINES += -DUSE_UTIL_DIRECTLY
 USE_UTIL_DIRECTLY = 1
 
+EXTRA_SHARED_LIBS +=    -Wl,-R${PREFIX}/lib/${MOZILLA_PKG_NAME}
